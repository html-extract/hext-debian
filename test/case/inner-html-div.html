<!DOCTYPE html>
<html>
  <head>
    <title>title</title>
    <meta charset="utf-8" />
  </head>
  <body>
    <div class="asd">
      <!-- comment -->
      <img alt="test" src="1.png" />
    </div>
  </body>
</html>
