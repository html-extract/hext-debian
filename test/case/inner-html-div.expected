{"html":"\n      <!-- comment -->\n      <img alt=\"test\" src=\"1.png\"/>\n    "}
